### PR TITLE
NPM testing 

### PR DESCRIPTION
**Pull request by ColinMcNeil**

### CODE REVIEW

File: prompts/npm/010_system_prompt.md [hunk 1]
```diff
@@ -1,47 +1,41 @@
...
-  The project has a node root package.json at {{path}} with the contents {{content}}.
-  Because there is already a root, the project does not need to be converted to npm.
-{{/project.node_roots}}
-{{^project.node_roots}}
-  The project does not have a node root, so the user should run `npm init`
-{{/project.node_roots}}
+Add a block to cd into the node root
 
+```sh
+cd $node_root
```
Comment (slimslenderslacks) [inline prompts/npm/010_system_prompt.md:23]: I don't see how these are actionable without an underlying kernel that can maintain this between code blocks. Maybe this is just something we have to track as potential feedback and look for signals that users want us to solve this?

File: prompts/npm/scripts/payload.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
+{
```
Comment (slimslenderslacks) [inline prompts/npm/scripts/payload.json:1]: Oh wait, this is not ready to merge yet is it?  This is still the version with hard coded node roots.

**Review by slimslenderslacks — CHANGES_REQUESTED** [review]
I thought this was in a reviewable state but you're still working on the npm extraction function, right?

File: prompts/npm/scripts/payload.json (hunk 2 above)
Comment (ColinMcNeil) [reply prompts/npm/scripts/payload.json:1]: This is now ready for review.

**Pull request merged**